components/integration/base/delete-snapshots.yaml

```
@@ -0,0 +1,25 @@
+kind: ClusterRole
+apiVersion: authorization.openshift.io/v1
```

The CI is failing because ArgoCD is syncing in a loop. I deployed this PR to my test cluster and I reproduce the issue. Looks like kustomize does not handle well the "authorization.openshift.io/v1" api as it is adding the namespace to the ClusterRole even though it should not as this is a global resource. This is causing the sync loop...

I missed this at review but you need to use:

```suggestion
rbac.authorization.k8s.io/v1
```

---

Sorry, my suggestion was not ok, it should have been: 
```suggestion
apiVersion: rbac.authorization.k8s.io/v1
```